src/ansi-c/ansi_c_entry_point.cpp

```diff
@@ -291,7 +321,7 @@ bool ansi_c_entry_point(
   const code_typet::parameterst &parameters=
     to_code_type(symbol.type).parameters();
 
-  if(symbol.name==standard_main)
+  if(symbol.name==ID_main)
```

Comment:
Well initially simply as I hadn't passed it in as a parameter, but then when I dug around to see how this parameter was being supplied and in multiple places was passing in as hard coded string `"main"` ([1](https://github.com/diffblue/cbmc/blob/master/src/ansi-c/ansi_c_language.cpp#L197), [2](https://github.com/diffblue/cbmc/blob/master/src/goto-cc/compile.cpp#L410)), this seemed preferable.